lib/web/ui/server.go

```diff
@@ -57,7 +57,18 @@ type Server struct {
 	// SSHLogins is the list of logins this user can use on this server
 	SSHLogins []string `json:"sshLogins"`
 	// AWS contains metadata for instances hosted in AWS.
-	AWS *types.AWSInfo `json:"aws,omitempty"`
+	AWS AWSMetadata `json:"aws,omitempty"`
```

Comment:
We should probably use a pointer here, otherwise we will get and `"aws": {}` instead of omitting the key entirely.

```suggestion
	AWS *AWSMetadata `json:"aws,omitempty"`
```